app/components/action-feed/template.hbs

```diff
@@ -1,4 +1,4 @@
-{{#if actionsList.isPending}}
+{{#if listLoading}}
```

Comment:
Instead of hiding the list while it's loading the next page, I think it'd be nicer to simply hide the "See more" link and show the loading indicator below the list of actions.

Reply:
this is handled now..thanks 👍